manageiq-providers-amazon.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,config.lib}/**/*"]
 
-  s.add_dependency("aws-sdk", ["~>2.8.7"])
+  s.add_dependency("aws-sdk", ["~>2.9.7"])
 
```

Comment:
@jerryk55 - do you still need this given that https://github.com/ManageIQ/manageiq-providers-amazon/pull/252 was merged?